tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -1121,3 +1124,133 @@ def test_workflows_halts_on_multiple_exact_matches(workflow_app):
 
     assert obj.status == ObjectStatus.HALTED
     assert obj.extra_data['_action'] == 'resolve_multiple_exact_matches'
+
+
+def fake_validation(data, schema=None):
+    return
```

Comment:
why we need that?

Reply:
To skip first validation and check if second is failing as it should.